backend/testfiles/execution/stdlib/json.dark

```diff
@@ -136,9 +136,80 @@ Json.parse<Float> "1e3" = PACKAGE.Darklang.Stdlib.Result.Result.Ok 1000.0 // PAC
 Json.parse<Float> "-1e3" = PACKAGE.Darklang.Stdlib.Result.Result.Ok -1000.0 // PACKAGE.Darklang.Stdlib.Result.Result.Ok?
 
 // TODO: test the upper/lower bounds
+Json.serialize<Float> 3.40282347e+38 = PACKAGE.Darklang.Stdlib.Result.Result.Ok
+  "3.40282347e+38"
```

Comment:
I don't think this is expected behavior - this 'works' and is desired-ish, but we have nothing implemented to really support this - we're just getting .NET things for free. Which is great, but won't work when we abandon the F# parser eventually. So, I'm not sure what to do here.

Reply:
(I'm specifically referring to:
- being able to parse things with `[number]e+[pow]`
- serializing with that same format)

Reply:
I agree. I'd suggest we remove the test.